apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -13,38 +13,23 @@ export const COLUMNS = {
   NOT_ANSWERED: 3,
 };
 
-const answerOptionsOneFormatter = (percentAnsweredOptionOne, {rowData}) => {
-  return (
-      <MultipleChoiceAnswerCell
-        id={rowData.id}
-        percentValue={rowData.percentAnsweredOptionOne}
-        isCorrectAnswer={rowData.optionOneIsCorrect}
-      />
-  );
-};
-
-const answerOptionsTwoFormatter = (percentAnsweredOptionOne, {rowData}) => {
-  return (
-      <MultipleChoiceAnswerCell
-        id={rowData.id}
-        percentValue={rowData.percentAnsweredOptionTwo}
-        isCorrectAnswer={rowData.optionTwoIsCorrect}
-      />
-  );
-};
+const alphabetMapper =  ['-', 'A', 'B', 'C', 'D', 'E', 'F', 'Not Answered'];
 
-const notAnsweredFormatter = (percentAnsweredOptionOne, {rowData}) => {
+const answerOptionsFormatter = (percentAnswered, {rowData, columnIndex, rowIndex, property}, index) => {
   return (
       <MultipleChoiceAnswerCell
         id={rowData.id}
-        percentValue={rowData.notAnswered}
+        percentValue={rowData.answers[columnIndex] && `${rowData.answers[columnIndex].percentAnswered}%` || '-'}
+        isCorrectAnswer={rowData.answers[columnIndex] && rowData.answers[columnIndex].isCorrectAnswer || false}
       />
   );
 };
 
 const questionAnswerDataPropType = PropTypes.shape({
   id: PropTypes.number.isRequired,
   question: PropTypes.string,
+  percentAnswered: PropTypes.string,
+  isCorrectAnswer: PropTypes.bool,
   percentAnsweredOptionOne: PropTypes.string,
```

Comment:
looks like some of these can be removed now.

Reply:
Fixed, Thanks!